<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="1">
     <widget class="QPushButton" name="button_filter_sobel">
      <property name="text">
       <string>Sobel Filter</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="button_filter_gauss">
      <property name="text">
       <string>Gauss Filter</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="button_filter_madian">
      <property name="text">
       <string>Median FIlter</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="button_filter_emboss">
      <property name="text">
       <string>Emboss Filter</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QLabel" name="label_image">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>336</width>
        <height>396</height>
       </size>
      </property>
      <property name="text">
       <string>You'll see the original image here!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="button_filter_sharpening">
      <property name="text">
       <string>Sharpening Filter</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="button_filter_blur">
      <property name="text">
       <string>Blur Filter</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="button_load_image">
      <property name="text">
       <string>Load Image</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="7">
     <widget class="QLabel" name="label4output">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>295</width>
        <height>396</height>
       </size>
      </property>
      <property name="text">
       <string>You'll see the output image here!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="9">
     <widget class="QPushButton" name="button_filter_compression">
      <property name="text">
       <string>Compression</string>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <widget class="QPushButton" name="button_save_image">
      <property name="text">
       <string>Save Image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
